### DISCUSSION TITLE
Can I use a recipe to fine tune a model ?

### DISCUSSION THREAD

**Discussion by Ca-ressemble-a-du-fake** [coqui-ai/TTS#1173] [General Q&A] [answered]
Hi,

I am trying to fine tune the Exp3 model (En-PT-Fr). So far I used the VITS recipe, customized it and launched the fine tuning from the downloaded checkpoint :

`
!python3 /content/drive/MyDrive/YourTTS/TTS/configs/vits_ljspeech.py \
--restore_path /content/drive/MyDrive/YourTTS/Models/Experiment3/best_model_latest.pth.tar
`
My `vits_ljspeech.py` has `vitsargs` (model args), `datasets`, ...  set up, 

But watching @thorstenMueller 's [tutorial](https://www.youtube.com/watch?v=O6KxJR95WpE) where he is using `config_path path/to/config.json` I now wonder if I am using the recipe the right way. 

Is there any difference between launching the training from a recipe (ie a custom python script) and launching it from `train_vits.py` with custom `config.json` ? Which way should I choose to fine tune a model ?

Thanks

**Comment by erogol** [comment, marked as answer]
Doesn' t matter how you run the code. What matters is the config fields, arguments. You should also use the --restore_model flag to point to the model you want to fine-tune.

**Reply by Ca-ressemble-a-du-fake** [reply]
Thank you @erogol , I'll have a look at --restore_model flag!
